apps/parser/.eslintignore

```diff
@@ -0,0 +1,11 @@
+node_modules
+.vscode
+.DS_Store
+/docs
+/coverage
+/lib
+/esm
+/cjs
+/browser
+esm
```

Comment:
I understand you want to ignore `esm` directory and all it's files. So `/esm` should be enough. Avoid repeating patterns.

Reply:
ok added.

Reply:
done.